zoo-css-challenge/style.css

```diff
@@ -40,31 +43,40 @@ h4,
 h5 {
   /*Purple*/
   color: #6320ee;
-  margin-bottom: 1em;
+  margin-bottom: 10px;
 }
 
 h1 {
-  font-size: 250%;
+  font-size: 1rem;
 }
 
 h2 {
-  font-size: 175%;
+  font-size: 1rem;
+  color: rgb(209, 66, 22);
+  text-align: center;
 }
 
 h3 {
-  font-size: 150%;
+  font-size: 1rem;
+  color: rgb(235, 15, 206);
+  text-align: center;
 }
 
-button {
+button:hover {
```

Comment:
If we change this from `button` to `button:hover`, it means we only show these button styles when we hover on a button which is not what we want. Instead, we can create a new rule `button:hover` and leave this `button {` alone